.github/workflows/ci.yml

```diff
@@ -26,39 +38,51 @@ jobs:
         uses: ruby/setup-ruby@v1
         with:
           ruby-version: ${{ matrix.ruby }}
-
-      - name: Setup gems
-        run: bundle install
-
-      - name: Rubocop
-        if: matrix.build_type == 'lint'
-        run: bundle exec rubocop
+          bundler-cache: true
 
       - name: Setup redis
-        if: matrix.build_type == 'redis'
         run: sudo apt-get install redis-server
 
       - name: Redis specs
-        if: matrix.build_type == 'redis'
         run: bin/rspec redis
 
-      - name: Setup test app gems
-        if: matrix.build_type == 'active_record'
-        run: cd spec/support/dummy_app && bundle install
+  active_record:
+    runs-on: ubuntu-latest
+    name: 'ActiveRecord ~>${{ matrix.rails }} (Ruby ${{ matrix.ruby }})'
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: ['2.7', '3.0', '3.1', '3.2']
+        rails: ['6.1.0']
```

Comment:
Can you help me understand why this is set to 6.1.0 when our Gemfile is 7.0.4.3?

Reply:
core is on rails 7, rails_failover supported (officially) only rails 6.x and did not test 7.0. I'm fixing deprecations and adding 7.x support in a separate PR (https://github.com/discourse/rails_failover/pull/48)